_templates/plugin/create/CHANGELOG.md.t

```diff
@@ -0,0 +1,14 @@
+---
+to: packages/<%= h.changeCase.paramCase(name) %>/CHANGELOG.md
+---
+<% pluginName = h.changeCase.paramCase(name) %>
+<% if (!locals.version){ locals.version = '0.0.1'} %>
```

Comment:
Radky 4 a 5 zpusobuji to, ze se obevi newline na zacatku souboru. To se da vyresit tak, ze se tenhle templatovaci retezec uzavre pomoci `-%>`.